File: go calc/calc.go
package main

import "fmt"

func main() {
	var num1 int16
	var num2 int16
	var sign string

	fmt.Println("pls enter your first number")
	fmt.Scan(&num1)
	fmt.Println("pls enter your second number")
	fmt.Scan(&num2)
	fmt.Println("pls enter your sign")
	fmt.Scan(&sign)

	if sign == "+" {
		var res int16 = num1 + num2
		fmt.Printf("your answer is %v ", res)
	}
	if sign == "-" {
		var res int16 = num1 - num2
		fmt.Printf("your answer is %v ", res)
	}
	if sign == "*" {
		var res int16 = num1 * num2
		fmt.Printf("your answer is %v ", res)
	}
	if sign == "/" {
		var res int16 = num1 / num2
		fmt.Printf("your answer is %v ", res)
	}

}

// go rating 10👍/10

File: kotlin calc/calc.kt
import java.util.Scanner

fun main(args: Array<String>) {

    val reader = Scanner(System.`in`)
    println("pls enter your first number: ")
    var num1:Int = reader.nextInt()
    println("pls enter your second number: ")
    var num2:Int = reader.nextInt()
    println("pls enter your sign: ")
    val sign = readLine()!!

    if (sign == "+"){
        var res:Int = num1 + num2
        println("your answer is $res")
    }
    if (sign == "-"){
        var res:Int = num1 - num2
        println("your answer is $res")
    }
    if (sign == "*"){
        var res:Int = num1 * num2
        println("your answer is $res")
    }
    if (sign == "/"){
        var res:Int = num1 / num2
        println("your answer is $res")
    }
}
// kotlin rating 9🤖/10